package.json

```diff
@@ -35,14 +35,16 @@
   "devDependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
     "@edx/browserslist-config": "1.2.0",
-    "@openedx/frontend-build": "14.0.3",
-    "@openedx/paragon": "22.3.2",
+    "@openedx/frontend-build": "github:edunext/frontend-build#dcoa/design-tokens-support",
+    "@openedx/paragon": "npm:@openedx/paragon@^22.0.0-alpha.25",
```

Comment:
It might worth to wait for the [Paragon sync PR](https://github.com/openedx/paragon/pull/3044) to be merged, so we won't downgrade paragon requirements, and still will be able to use the latest version of paragon.